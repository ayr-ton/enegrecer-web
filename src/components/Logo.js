import React from 'react';

const Logo = () => (
  <span className="en-logo">Enegrecer</span>
);

export default Logo;
