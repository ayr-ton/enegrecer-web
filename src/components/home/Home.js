import React, { Component } from 'react'
import { Link } from 'react-router-dom'

export default class Home extends Component {
    render() {
        return (
            <div id="home">
                <h1>Enegrecer</h1>
                <Link to="/categorias">Categorias</Link>
            </div>
            
        );
    }
}

