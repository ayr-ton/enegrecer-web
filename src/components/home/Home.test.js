import React from 'react';
import { shallow } from 'enzyme';

import Home from './Home';
import Logo from './../Logo';

describe('Home', () => {
  const wrapper = shallow(<Home />);

  it('tem um topbar', () => {
    expect(wrapper.find('Topbar').exists()).toBeTruthy();
  });

  it('tem um topbar com a logo na esquerda', () => {
    expect(wrapper.find('Topbar').props().leftComponent).toEqual(<Logo />);
  });
});
