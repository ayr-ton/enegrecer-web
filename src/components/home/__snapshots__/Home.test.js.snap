// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home DOM 1`] = `
<div>
  <div
    id="banner"
  >
    <div
      id="banner__blur"
    />
    <div
      className="container"
    >
      <Topbar
        leftComponent={<Logo />}
        rightComponent={
          <Nav
            history={
              Object {
                "push": [Function],
              }
            }
          />
        }
      />
      <div
        className="row"
      >
        <div
          className="col-md-4"
        >
          <div
            className="en-subtitle--primary"
          >
            LOREM IPSUM SIT
          </div>
          <div
            className="en-title en-title--light en-title--marker"
          >
            LOREM IPSUM CONSECTETUR
          </div>
          <p
            className="en-text--light"
            id="en-banner__description"
          >
            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
          </p>
          <Botao
            colored={true}
            onClick={[Function]}
            outline={false}
            primary={false}
          >
            Denunciar Racismo
          </Botao>
        </div>
      </div>
    </div>
  </div>
  <Footer>
    <Social
      networks={
        Array [
          Object {
            "href": "#",
            "icon": "facebook-square",
            "title": "Facebook",
          },
          Object {
            "href": "#",
            "icon": "twitter",
            "title": "Twitter",
          },
          Object {
            "href": "#",
            "icon": "instagram",
            "title": "Instagram",
          },
        ]
      }
    />
  </Footer>
</div>
`;
